src/NHSDPublicBrowseAcceptanceTests.Tests/Utils/EnvironmentVariables.cs

```diff
@@ -60,8 +70,23 @@ private static string JsonConfigValues(string section, string defaultValue)
 
             return string.IsNullOrEmpty(result) ? defaultValue : result;
         }
-    }
 
+        private static string DefaultUri()
+        {
+            var uri = "https://docker.for.win.localhost";
```

Comment:
for me, this doesn't work with https
If i use http then it works fine
I also don't understand why you need to be playing with the host?

Reply:
all of that will be removed soon (when i have finalised the configuration)

Reply:
URI is changing in the next commit